apps/platform-app/src/app/[locale]/(profile)/(routes)/my-profile/(components)/CreateProfile/WorkInformation/WorkInformation.tsx

```diff
@@ -65,6 +67,9 @@ const WorkInformation = () => {
       )
     }
```

Comment:
**issue**: Ensure that the `onClick` handler is not called unnecessarily. Consider using a memoized callback to prevent unnecessary re-renders.